app/scripts/analysis/tools/powercalc/Powercalc.module.coffee

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+Module = require 'scripts/BaseClasses/BaseModule.coffee'
+
+module.exports = powercalc = new Module
+
+  # module id for registration
+  id: 'app_analysis_powercalc'
+
+  # module components
+  components:
+    services:
+      'app_analysis_powercalc_msgService': require 'scripts/analysis/tools/powercalc/PowercalcMsgService.service.coffee'
+      'app_analysis_powercalc_algorithms': require 'scripts/analysis/tools/powercalc/PowercalcAlgorithms.service.coffee'
+      'app_analysis_powercalc_initService': require 'scripts/analysis/tools/powercalc/PowercalcInit.service.coffee'
+      'app_analysis_powercalc_dataService': require 'scripts/analysis/tools/powercalc/PowercalcDataService.service.coffee'
+      'app_analysis_powercalc_TwoTGUI': require 'scripts/analysis/tools/powercalc/PowercalcTwoTGUI.service.coffee'
```

Comment:
name convention for components names is camelCase, i.e. it should be `'app_analysis_powercalc_twoTGui'` or even `'app_analysis_powercalc_twoTTest'` because service itself doesn't really provide any GUI